lawnchair/src/app/lawnchair/ui/preferences/PreferencesDashboard.kt

```diff
@@ -176,6 +189,27 @@ fun PreferencesDebugWarning() {
     }
 }
 
+@Composable
+fun PreferencesSetDefaultLauncherWarning() {
+    val context = LocalContext.current
+    Surface(
+        modifier = Modifier.padding(horizontal = 16.dp),
+        shape = MaterialTheme.shapes.large,
+        color = Material3Theme.colorScheme.errorContainer
```

Comment:
Error color seems a little too harsh, maybe something like `surface` or similar can work.

Reply:
Yeah, I was seeking some colors like yellow but didn't decide to use which one.

Reply:
cf609c07ffb93d59d62cb40e35c3e257c4da7f2d